keyboards/ph_arisu/info.json

```diff
@@ -0,0 +1,12 @@
+        "LAYOUT": {
+            "layout": [{"label":"2,7", "x":17.8, "y":0.9}, {"label":"0,1", "x":3.7, "y":0.9500000000000001}, {"label":"1,5", "x":13.15, "y":0.9500000000000001}, {"label":"0,0", "x":1.7, "y":1.0}, {"label":"1,0", "x":2.7, "y":1.0}, {"label":"0,6", "x":14.149999999999999, "y":1.0}, {"label":"1,6", "x":15.149999999999999, "y":1.0}, {"label":"0,7", "x":16.15, "y":1.0}, {"label":"4,7", "x":18.1, "y":1.9000000000000001}, {"label":"2,5", "x":13, "y":1.9500000000000002}, {"label":"2,0", "x":1.5, "y":2.0, "w":1.5}, {"label":"3,0", "x":3, "y":2.0}, {"label":"3,5", "x":14, "y":2.0}, {"label":"2,6", "x":15, "y":2.0}, {"label":"3,6", "x":16, "y":2.0, "w":1.5}, {"label":"6,7", "x":18.4, "y":2.9000000000000004}, {"label":"4,0", "x":1.3, "y":3.0, "w":1.75}, {"label":"5,0", "x":3.05, "y":3.0}, {"label":"4,5", "x":13.4, "y":3.0}, {"label":"5,5", "x":14.4, "y":3.0}, {"label":"4,6", "x":15.4, "y":3.0, "w":2.25}, {"label":"6,0", "x":1.05, "y":4.0, "w":2.25}, {"label":"7,0", "x":3.3, "y":4.0}, {"label":"7,4", "x":13.100000000000001, "y":4.0}, {"label":"6,5", "x":14.100000000000001, "y":4.0}, {"label":"7,5", "x":15.100000000000001, "y":4.0, "w":1.75}, {"label":"7,6", "x":17.5, "y":4.25}, {"label":"8,0", "x":1.05, "y":5.0, "w":1.5}, {"label":"8,6", "x":16.5, "y":5.25}, {"label":"9,6", "x":17.5, "y":5.25}, {"label":"8,7", "x":18.5, "y":5.25}, {"label":"1,1", "x":5.05, "y":1.66533453693e-16}, {"label":"0,2", "x":6.05, "y":1.66533453693e-16}, {"label":"1,2", "x":7.05, "y":1.66533453693e-16}, {"label":"0,3", "x":8.05, "y":1.66533453693e-16}, {"label":"2,1", "x":4.6, "y":1.0000000000000002}, {"label":"3,1", "x":5.6, "y":1.0000000000000002}, {"label":"2,2", "x":6.6, "y":1.0000000000000002}, {"label":"3,2", "x":7.6, "y":1.0000000000000002}, {"label":"4,1", "x":4.85, "y":2.0}, {"label":"5,1", "x":5.85, "y":2.0}, {"label":"4,2", "x":6.85, "y":2.0}, {"label":"5,2", "x":7.85, "y":2.0}, {"label":"6,1", "x":5.3, "y":3.0}, {"label":"7,1", "x":6.3, "y":3.0}, {"label":"6,2", "x":7.3, "y":3.0}, {"label":"7,2", "x":8.3, "y":3.0}, {"label":"8,2", "x":6.6, "y":4.0, "w":2}, {"label":"9,2", "x":8.6, "y":4.0, "w":1.25}, {"label":"8,1", "x":5.05, "y":4.05, "w":1.5}, {"label":"1,3", "x":8.45, "y":3.6}, {"label":"0,4", "x":9.45, "y":3.6}, {"label":"1,4", "x":10.45, "y":3.6}, {"label":"0,5", "x":11.45, "y":3.6}, {"label":"2,3", "x":8.05, "y":4.6000000000000005}, {"label":"3,3", "x":9.05, "y":4.6000000000000005}, {"label":"2,4", "x":10.05, "y":4.6000000000000005}, {"label":"3,4", "x":11.05, "y":4.6000000000000005}, {"label":"4,3", "x":8.2, "y":5.6000000000000005}, {"label":"5,3", "x":9.2, "y":5.6000000000000005}, {"label":"4,4", "x":10.2, "y":5.6000000000000005}, {"label":"5,4", "x":11.2, "y":5.6000000000000005}, {"label":"6,3", "x":8.75, "y":6.6000000000000005}, {"label":"7,3", "x":9.75, "y":6.6000000000000005}, {"label":"6,4", "x":10.75, "y":6.6000000000000005}, {"label":"9,3", "x":7.75, "y":7.6000000000000005, "w":2.75}, {"label":"9,4", "x":10.55, "y":7.65, "w":1.5}]
```

Comment:
You used the KLE -> QMK Configurator converter, right?  
Configurator does not currently support rotated keys, that's why this json is full of rounding errors. Also it doesn't really render properly in Configurator. (You can check by pressing Ctrl+Shift+I on Configurator and loading the json.)

Check the Alice's info.json to see how this should be fixed.

Reply:
Just noticed that https://github.com/qmk/qmk_firmware/tree/master/keyboards/arisu also misses this info.json. Once you figure it out, you can probably contribute the more-or-less same json for both boards.

Reply:
Noted Many thanks.

Reply:
Updated info.json https://github.com/qmk/qmk_firmware/pull/9612/commits/34a8366fb228030b679fbeb73cff37706133af27